CRM/Member/BAO/Membership.php

```diff
@@ -2700,4 +2705,53 @@ public static function mergeMemberships($mainContactID, $otherContactID, &$sqlQu
     $sqlQueries = array_merge($sqlQueries, $newSql);
   }
 
+  /**
+   * Returns array with recurring contributions used to pay for membership.
+   *
+   * @param $membershipID
+   *
+   * @return array
+   */
+  public static function getRecurContributions($membershipID) {
+    $query = "
```

Comment:
Is there no API w could use here? Also i don't see `contribution_count` being used om the `$recurringContributions` array

Reply:
I could use chained API calls, but other than that, I don't think there is a way to get recurring contributions tied to a membership.  I'll remove the `contribution_count` from the query, though.